tests/fixtures/next-features.js

```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
+
+export function Component() {
+  const ref = React.useRef();
+
+  React.useEffect(() => {
+    ref.current?.focus();
+  });
+
+  return React.createElement("div", { ref });
+}
```

Comment:
I would prefer that we rename this to something like es2020-features.js as I would interpret "next" as referring to the future.